src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+
+[Sharding](/upgrades/shard-chains/) is a  proposed method for increasing data throughput on Ethereum’s execution layer. In sharding, the network is split into a select number of sub-chains—each with a dedicated set of validators. 
+
+Validators will only need to run full nodes for their shard and run in light-client capacity for other shards. Sharding increases data space available to rollups since the job of storing data is split across different shards. 
+
+But data sharding introduces a new problem: “What if validators on one shard become malicious and start processing invalid state transitions?”. This is possible since full nodes no longer have access to the same transaction data as is currently the case. 
+
+Implementing data sharding requires creating a system for nodes to verify data availability in other shards without downloading blocks. If validators must download data dumped on every shard, then the purpose of sharding is defeated. 
+
+## Types of data availability systems {#types-of-data-availability-systems} 
+
+### On-chain data availability {#on-chain-data-availability} 
+
+The standard solution to solving data availability is to force block producers to publish all transaction data and have validating nodes download it. Ethereum uses this method, as do rollups (and other scaling solutions) that post transaction data on the L1 blockchain. 
+
+The problem is that on-chain data availability places bottlenecks on scalability. Full nodes have to download the same data and check it, which reduces the number of transactions that can be processed per second.
+
+More importantly, on-chain DA has negative implications for decentralization since it increases the data that full nodes must hold. If Ethereum’s state spirals, validators must invest in more expensive machines—which would probably reduce the number of people willing to run a validating node.  
+
+### Off-chain data availability {#off-chain-data-availability} 
+
+Off-chain data availability solutions move data storage off Ethereum's base layer. The block producer doesn't publish the data on-chain, but provides a cryptographic commitment to prove the availability of the data. 
+
+This is a method used by some scaling solutions, such as [validiums](/developers/docs/scaling/validium/), to reduce the amount of data posted on Ethereum. The drawback is that this system requires trusting block producers to release the data if necessary.  
+
+## What are some solutions to the data availability problem? {#solutions-to-data-availability-problem}
+
+As mentioned, the data availability problem concerns the ability to verify availability of the transaction data for a newly proposed block. Solutions to this problem employ some mechanism for guaranteeing data availability. 
```

Comment:
Can we order the subheadings here from most to least impactful from the Ethereum ecosystem's perspective? In my mind data availability sampling is the favourite atm?

Reply:
Fixed this.